scripts/releasing/patch-trigger.js

```diff
@@ -80,13 +80,19 @@ async function main() {
   console.log(`Processing patch trigger for branch: ${headRef}`);
 
   // Extract base version and channel from hotfix branch name
```

Comment:
Consider extracting all of this into a new func that has a nice doc string and maybe a nice return interface to sort of abstract this a bit and make it very clear.